spec.md

```diff
@@ -1050,6 +1050,11 @@ message Volume {
   // COs MAY use this information along with the topology information
   // returned by NodeGetInfo to ensure that a given volume is accessible
   // from a given node when scheduling workloads.
+  // The volume is accessible from all locations in the
+  // specified topology.
+  // The volume is accessible from a node if the volume's
```

Comment:
... from a node if at least one of the volume's accessible topologies is a superset of ...

because a volume can have multiple accessible topologies

Reply:
I've added "union of", since I am already using the term of set theory.